test/libethereum/BlockChainTests.cpp

```diff
@@ -34,10 +34,13 @@ using namespace json_spirit;
 using namespace dev;
 using namespace dev::eth;
 
+
 namespace dev {
 
 namespace test {
 
+eth::Network TestBlockChain::sealEngineNetwork = eth::Network::FrontierTest;
```

Comment:
Also, currently you have `FrontierTest`  and `HomesteadTest` . Which are not enough IMO. Frontier is everthing according to Frontier rules, and Homestead for Homestead rules, but where do you have the switch?
I introduced simple `Tests` where everything before block X is Frontier, and after it is Homestead. I think you should use that.